src/pocketmine/tile/FlowerPot.php

```diff
@@ -66,12 +65,12 @@ public function canAddItem(Item $item) : bool{
 	}
 
 	public function getItem() : Item{
-		return Item::get((int) ($this->namedtag["item"] ?? 0), (int) ($this->namedtag["mData"] ?? 0), 1);
+		return Item::get($this->namedtag->item->getValue(), $this->namedtag->mData->getValue(), 1);
```

Comment:
In case you were thinking why I did not check whether `$this->namedtag->item` or `$this->namedtag->mData` is set before using `getValue()`, see https://github.com/Muqsit/PocketMine-MP/blob/b7100070e2b2071a1b38579d337362e579dfbef9/src/pocketmine/tile/FlowerPot.php#L35-#L40